docs/getting-started/github-apps/github-apps.rst

```diff
@@ -6,6 +6,12 @@ Automatic Syncing with GitHub Apps and .dockstore.yml
 
 Overview
 --------
+Summary of GitHub App usage:
+
+- Install Dockstore GitHub App
+- Push a new commit with a .dockstore.yml file present
```

Comment:
This seems too subtle and indirect. 

How about something more like "After installing the github app, make sure to push a new commit to a branch or create a tag with a .dockstore.yml file."

Reply:
the important box is for important notes, I don't think that's appropriate for summaries. "After installing the GitHub app" is already mentioned in the point above it.  "create a tag with a .dockstore.yml" seems like an edge case in which the person already has a .dockstore.yml present and doesn't need to make another commit.

Reply:
Changed to numbered list to indicate that it's a sequential process. The summary is already placed at the top of the page, if they decide to skip that, then I don't think there's anything we can do that would make a difference

Reply:
putting this inside the note box as mentioned with the use case in https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html . seems to make sense

Reply:
> the important box is for important notes, I don't think that's appropriate for summaries 

Fair, I'm ok with rewording it to not be a summary and to make it an important note.
Looking at the preview, the numbered list doesn't stand out at all